packages/@aws-cdk/aws-cloudwatch/test/integ.alarm-and-dashboard.ts

```diff
@@ -102,6 +102,10 @@ dashboard.addWidgets(new cloudwatch.SingleValueWidget({
   metrics: [sentMessageSizeMetric],
   fullPrecision: true,
 }));
+dashboard.addWidgets(new cloudwatch.ContributorInsightsWidget({
```

Comment:
Just to be pedantic. This test is just to test the integration of an insight widget?